Explorer/Assets/DCL/MapRenderer/MapLayers/IMapLayerController.cs

```diff
@@ -16,6 +16,8 @@ void IMapLayerController.SetParameter(IMapLayerParameter mapLayerParameter) =>
 
     internal interface IMapLayerController : IDisposable
     {
+        UniTask InitializeAsync(CancellationToken cancellationToken);

```

Comment:
I see that many implementation classes have nothing to initialize. It is pending to implement? Otherwise i would propose to explicitly call this method to the class that requires it instead of adding it into the interface

Reply:
currently only 3 layers don't use it, but one is pending the integration, I would leave it like this :)